substrate/frame/support/src/lib.rs

```diff
@@ -177,8 +177,7 @@ pub use frame_support_procedural::storage_alias;
 
 pub use frame_support_procedural::derive_impl;
 
-/// Experimental macros for defining dynamic params that can be used in pallet configs.
-#[cfg(feature = "experimental")]
```

Comment:
FRAME is still using an experimental stage for its upcoming features, please revert.

Reply:
Yeah, we do want this in FRAME.

Reply:
I are not thinks! I works! I removes stuff!

Reply:
Reverted everything but Aura, sorry for misunderstanding :)